src/xx-ld-shas

```diff
@@ -2,91 +2,90 @@
+linux-armv7-ld-linux-armv7 702537f8df38d733420309a8506ea9f98ac9166f
+linux-armv7-ld-linux-ppc64le be373ede53d871051f8689a55dfac0053d970b05
+linux-armv7-ld-linux-s390x f6800e1cdb5407c6d5268fb5311cdb89462761ab
+linux-ppc64le-ld-linux-386 d57e4a6f4c65cbddf6b476fdb63efc6419a1a380
+linux-ppc64le-ld-linux-amd64 07a9e47ab1cf92502efdf3695e319de683eb8a7a
+linux-ppc64le-ld-linux-arm64 6693a54bbf8c1caa8c395edffd769d3d18261290
+linux-ppc64le-ld-linux-armv6 4bb1bbf3b57047366d02820496cf578a684c5f86
+linux-ppc64le-ld-linux-armv7 9056fa87bacc43524fc6345eaff305ad6cc68071
+linux-ppc64le-ld-linux-ppc64le 2b6dad76462045293adf6f614e6f4892495aef57
+linux-ppc64le-ld-linux-s390x 6fa3774a15a43373382d57962b7270730678f2b1
+linux-riscv64-ld-linux-386 79b2cbb722f55625a7d946959b5a7894db2dcdf6
+linux-riscv64-ld-linux-amd64 97beba5e0ed68c124e8a27c77508d9d9a7ad7019
+linux-riscv64-ld-linux-arm64 71a044d6852913a462fe43a880d3463b1607778f
+linux-riscv64-ld-linux-armv6 997afb5c6ae6a9c6abea4b78828dbcb379dc329e
+linux-riscv64-ld-linux-armv7 e59dc6075ad5efde663f01854de5d028d3952411
+linux-riscv64-ld-linux-ppc64le 7f80a6b69cf1317538a7d4ce272ca96852b2d52c
+linux-riscv64-ld-linux-s390x ecf7ae88ace997f488d3e06cf1179f9fb41851ca
+linux-s390x-ld-linux-386 239444215bf26a1bf27dc2e08d403eae487d8883
+linux-s390x-ld-linux-amd64 fb703382754033ed66d6a8a151c2d9b798a988e6
+linux-s390x-ld-linux-arm64 ab6b4e01233fe556362afe0b524878d2f98d1f97
+linux-s390x-ld-linux-armv6 c6f48b1c05b1681aedd541bf5644acd83170cf89
+linux-s390x-ld-linux-armv7 06a96ff9378303ed1b1c5a241ae68a2f9ce0e2e3
+linux-s390x-ld-linux-ppc64le 9f09a31c2b33fb2d7c13cb3c56a20da5e6d34e55
+linux-s390x-ld-linux-s390x 300961695d48096ac34152c916d75a88eb537eb2
+windows-386-ld-linux-386 2d77fc1682fdbe587fb27acefa244a8a7599f526
+windows-386-ld-linux-amd64 5b689d14620e7e40fdc50ae50cef09e46694f3e4
+windows-386-ld-linux-arm64 d54cf8094ce4655025db75bfc3dc15ea0718443f
+windows-386-ld-linux-armv6 9f6e4bfad6b0c0b4ec27bc10f3ec0c457e1f1b1e
+windows-386-ld-linux-armv7 f2969d14745003a718358de3c8e499eb7a78b6bc
+windows-386-ld-linux-ppc64le e5090c7b29c92caf9cc98f6b13ee643ff80cb7a5
+windows-386-ld-linux-s390x aac9f041f5a113690b0411f077f1865bbe99df39
+windows-amd64-ld-linux-386 89215f3cd30ff429a634fecb2f2a0618627505e2
+windows-amd64-ld-linux-amd64 0e55e95d1990555056003dbf892783563571a49f
+windows-amd64-ld-linux-arm64 bd8279dbf811cf2d4313ad50dcf646a224245190
+windows-amd64-ld-linux-armv6 e18cefe0e78913a2a0f56589319e16b0301c7c04
+windows-amd64-ld-linux-armv7 cb3dc911fac715d8220174777f29b6fd4a8eff8c
+windows-amd64-ld-linux-ppc64le 4d6bbe572070a570ea46ed45136bc96bc72b5cbb
+windows-amd64-ld-linux-s390x 31fa12ee50baa4defc6b4f7b232306c02f95d46a
 EOT
 )
 export LD_SHAS
 
 LD64_SHAS=$(
   cat <<'EOT'
-ld64-signed-linux-386 0cb9a9420ab2eb8311a06673f65e42e70f3dd1c2
-ld64-signed-linux-amd64 9e506cf2e77312a76f890568f60d0f61de1e1932
-ld64-signed-linux-arm64 d4e74cec34a0e46499d7c49b96f02407e0bd5275
-ld64-signed-linux-armv6 b93606174c563c4f7a6fcf0e9a02edbddb2c0985
```

Comment:
@tonistiigi Looks armv6 is missing, was it an oversight from https://github.com/tonistiigi/xx/pull/140/files#diff-870f6fe23fc034f008f5203ee1e628d7bfa65a49095d5a4688db498328449ed2L259?